style.css

```diff
@@ -18,6 +18,78 @@ body {
   font-weight: bold;
 }
 
+img {
+  width: 100%;
+}
+
+a {
+  text-decoration: none;
+}
+
+.col-5 a {
+  text-decoration: none;
+}
+
+#menupopup {
+  width: 100%;
+  height: 100%;
+  display: none;
+  background-color: #6070ff;
+  position: absolute;
+  top: 0;
```

Comment:
- Good job working on the Popup functionalities, however, I noticed some styling issues on your popup listed below.

     - Please the spacing on the heading of your popup is a bit too small, making it look squashed together. Please use appropriate spacing as in the Figma design. Image below:

            - **Your design**         
![image](https://user-images.githubusercontent.com/58771507/202106573-9c6df4f5-1509-4ee9-bbc5-bb45fec577b6.png)

            - **Figma Design**        
![image](https://user-images.githubusercontent.com/58771507/202106773-c61bb4c3-a43f-4a49-86cf-30ab79a04bd6.png)


   - Please add appropriate spacing to popup tags and buttons, also please include the horizontal line as in the Figma design
        - **Your Design**
        
![image](https://user-images.githubusercontent.com/58771507/202107261-0e958a2c-1e5b-4553-ba43-357856434b99.png)


         -  **Figma design**
  
![image](https://user-images.githubusercontent.com/58771507/202107535-efe8f2bd-4319-4ba1-93f1-316f79bc5263.png)



- Please also make use of the appropriate font-weight as it is on the Figma popup, currently, some of your text on the popup is too thick compared to that of the Figma design.

Please feel free to reply to this thread for clarification. Thanks

Reply:
@Ginohmk please you need to be connected to the internet before you can load the font weight, else the default will always be font-weight of 700;

Reply:
I connected to the internet, I always have my internet connections on, and all fonts are loaded properly but issues were with the font weight. You can share a screenshot of your popup, to be sure. Thanks for reaching out.